framework/src/Volo.Abp.BackgroundWorkers/Volo/Abp/BackgroundWorkers/AsyncPeriodicBackgroundWorkerBase.cs

```diff
@@ -35,22 +35,24 @@ public async override Task StopAsync(CancellationToken cancellationToken = defau
         }
 
         private void Timer_Elapsed(object sender, System.EventArgs e)
+        {
+            // Discard the result
+            _ = DoWorkAsync();
```

Comment:
All is fine, except this. Because `AbpTime` is designed to prevent overlaps (it prevents to execute `Timer_Elapsed` again if the previous work hasn't been completed). If we fire & forget, it causes overlap if execution duration is longer than the period.

The solution can be creating `AbpAsyncTimer` that internally solves this problem and use it in `AsyncPeriodicBackgroundWorkerBase.cs`.